man/mkdocs/mkdocs.yml

```diff
@@ -1,46 +1,316 @@
+    - navigation.tabs.sticky
+
+    # - content.action.edit # Edit on GitHub
   palette:
     primary: custom
-copyright: !ENV COPYRIGHT
+    # accent: custom
+  icon:
+    repo: fontawesome/brands/github
+    # edit: material/pencil
+    view: material/eye
+
+# Customization
+extra:
+  homepage: ./index.html
+  # We can add localized versions of the site here
+  # alternate:
+  #   - name: English
+  #     lang: en
+  #   - name: Deutsch
+  #     link: /de/
+  #     lang: de
+  #   - name: Español
+  #     link: /es/
+  #     lang: es
+  #   - name: Slovenčina
+  #     link: /sk/
+  #     lang: sk
+  #   - name: Italiano
+  #     link: /it/
+  #     lang: it
+
 extra_css:
   - grassdocs.css
+
+# Plugins
 plugins:
   - search
   - glightbox
-use_directory_urls: false
-nav:
-  - GUI: wxGUI.md
-  - Startup: grass.md
-  - Databases: database.md
-  - Display: display.md
-  - General: general.md
-  - Imagery: imagery.md
-  - Misc: miscellaneous.md
-  - Postscript: postscript.md
-  - Raster: raster.md
-  - Raster 3D: raster3d.md
-  - SQL: sql.md
-  - Temporal: temporal.md
-  - Variables: variables.md
-  - Vector: vector.md
-  - Keywords: keywords.md
-  - Topics: topics.md
+  # - social
+  # - tags
+  # - git-revision-date-localized:
+  #     enable_creation_date: true
+  # - git-committers:
+  #     repository: OSGeo/grass
+  #     branch: main
+
+# Markdown extensions
 markdown_extensions:
   - admonition
   - pymdownx.details
   - pymdownx.superfences
   - attr_list
   - md_in_html
+
+# Navigation
+nav:
+  - Interfaces:
+      - Shell:
+          - Overview: grass.md
+          # - Getting Started: grassintro.md # Does not exist yet
+          - Environmental Variables: variables.md
+      - GUI:
+          - Overview: wxguiintro.md
+          - Features: wxGUI.md
+          - Tool Dialogs: wxGUI.modules.md
+          - Attrribute Table Managment: wxGUI.dbmgr.md
+          - Cartographic Composer: wxGUI.psmap.md
+          - Data Catalog: wxGUI.datacatalog.md
+          - Vector Digitizer: wxGUI.vdigit.md
+          - Raster Digitizer: wxGUI.rdigit.md
+          - Graphical Modeler: wxGUI.gmodeler.md
+          - Ground Control Points Manager: wxGUI.gcp.md
+          - List of Components: wxGUI.components.md
+
+      # - Scripting:
+      #     - Overview: scriptingintro.md
+      #     - Python: python.md
+      #     - Shell Scripting: shell.md
+      #     - Batch Processing: batch.md
+      #     - R: r.md
+      # - Jupyter Notebook: jupyter.md
+
+  - Project Management:
+      - General Tools: general.md
+      - Projection and Transformations: projectionintro.md
+
+  - Processing Tools:
+      - Raster:
+          - Overview: rasterintro.md
+          - Raster Tools: raster.md
+      - Raster 3D:
+          - Overview: raster3dintro.md
+          - Raster 3D Tools: raster3d.md
+      - Vectors:
+          - Overview: vectorintro.md
+          - Vector Tools: vector.md
+      - GUI Tools:
+          - Network Analysis: wxGUI.vnet.md
```

Comment:
Something smart is needed here, but let's just move this for now to the large GUI tools section below.